src/users/__tests__/userSelectors-test.js

```diff
@@ -37,7 +37,15 @@ describe('getAccountDetailsUser', () => {
       },
       users: [],
     });
-    const expectedUser = { email: 'b@a.com', fullName: 'b@a.com', avatarUrl: '' };
+    const expectedUser = {
```

Comment:
Include this in a separate PR with the change to NULL_USER.

Reply:
@borisyankov sure will do.